File: src/app/browser/main/tab/page/navigation/bookmark.cpp
#include "bookmark.hpp"

using namespace app::browser::main::tab::page::navigation;

Bookmark::Bookmark()
{
    set_action_name(
        "tab.bookmark"
    );

    set_icon_name(
        "starred-symbolic" // | non-starred-symbolic
    );

    set_tooltip_text(
        _("Toggle bookmark")
    );
}

File: src/app/browser/main/tab/page/navigation/bookmark.rs
use gtk::Button;

pub fn new() -> Button
{
    let button = Button::builder()

        .icon_name(
            "starred-symbolic"
        )

        .tooltip_text(
            "Toggle bookmark"
        )

        .sensitive(
            false
        )

        .build();

    return button;
}